lib/index.js

```diff
@@ -125,9 +125,9 @@ internals.handler = function (route, handlerOptions) {
             request.info.remotePort &&
             request.info.remoteAddress) {
             options.headers['x-forwarded-for'] = (options.headers['x-forwarded-for'] ? options.headers['x-forwarded-for'] + ',' : '') + request.info.remoteAddress;
-            options.headers['x-forwarded-port'] = (options.headers['x-forwarded-port'] ? options.headers['x-forwarded-port'] + ',' : '') + request.info.remotePort;
-            options.headers['x-forwarded-proto'] = (options.headers['x-forwarded-proto'] ? options.headers['x-forwarded-proto'] + ',' : '') + request.server.info.protocol;
-            options.headers['x-forwarded-host'] = (options.headers['x-forwarded-host'] ? options.headers['x-forwarded-host'] + ',' : '') + request.info.host;
+            options.headers['x-forwarded-port'] = options.headers['x-forwarded-port'] ? options.headers['x-forwarded-port'] : request.info.remotePort;
```

Comment:
Could be all written like `options.headers['x-forwarded-port'] = options.headers['x-forwarded-port'] || request.info.remotePort;`.